versioned/persist/nontx/src/test/java/org/projectnessie/versioned/persist/nontx/NonTransactionalDatabaseAdapterConfigTest.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (C) 2022 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.projectnessie.versioned.persist.nontx;
+
+import org.junit.jupiter.api.Test;
+import org.projectnessie.versioned.persist.adapter.AdjustableDatabaseAdapterConfig;
+import org.projectnessie.versioned.persist.adapter.DatabaseAdapterConfig;
+import org.projectnessie.versioned.persist.tests.DatabaseAdapterTestUtils;
+
+public class NonTransactionalDatabaseAdapterConfigTest {
+
+  @Test
+  void allConfigPropertiesAreAdjustable() {
+    DatabaseAdapterTestUtils.assertAdjustableConfigConvention(
+        DatabaseAdapterConfig.class, AdjustableDatabaseAdapterConfig.class);
```

Comment:
due to the location of `DatabaseAdapterTestUtils` it is not easily possible to test this in `versioned.persist.adapter` module... open to ideas how to do this better but i thought redundantly testing this here should be fine.

Reply:
It's okay as it is.